src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -185,6 +189,24 @@ package object pings {
       } catch { case _: Throwable => None }
     }
 
+    // Return the number of values greater than threshold
+    def histogramThresholdCount(histogramName: String, threshold: Int, processType: String): Long = {
+      implicit val formats = org.json4s.DefaultFormats
+      val histogram = if(processType == "main") {
+        this.meta.`payload.histograms`
+      } else {
+        this.payload.processes \ processType \ "histograms"
+      }
+
+      histogram \ histogramName \ "values" match {
```

Comment:
I did create an updated function for this that covers more unsafe values in the data.